common/config/wv.json/layers/amsr2/AMSRU2_Sea_Ice_Concentration_12km.json

```diff
@@ -0,0 +1,16 @@
+{
+    "layers": {
+        "AMSRU2_Sea_Ice_Concentration_12km": {
+            "id":       "AMSRU2_Sea_Ice_Concentration_12km",
+            "title":    "Sea Ice Concentration (12 km)",
+            "subtitle": "GCOM-W1 / AMSR2",
```

Comment:
Should these subtitles be "AMSRU2" instead of "AMSR2"?

Reply:
No - it's confusing, but correct.